docs/reference/objectmodel/requirement-set-1.7/outlook-requirement-set-1.7.md

```diff
@@ -35,7 +35,7 @@ Requirement set 1.7 includes all of the features of [Requirement set 1.6](../req
 - Added [Office.MailboxEnums.RecurrenceTimeZone](/javascript/api/outlook_1_7/office.mailboxenums.recurrencetimezone): Adds a new enum that specifies the time zone applied to the recurrence.
 - Added [Office.MailboxEnums.RecurrenceType](/javascript/api/outlook_1_7/office.mailboxenums.recurrencetype): Adds a new enum that specifies the type of recurrence.
 - Added [Office.MailboxEnums.WeekNumber](/javascript/api/outlook_1_7/office.mailboxenums.weeknumber): Adds a new enum that specifies the week of the month.
-- Modified [Office.EventType](/javascript/api/office/office.eventtype): Modifies to support RecurrenceChanged, RecipientsChanged, and AppointmentTimeChanged events through addition of `RecurrenceChanged`, `RecipientsChanged`, and `AppointmentTimeChanged` entries respectively.
+- Modified [Office.EventType](/javascript/api/office/office.eventtype): Adds support for RecurrenceChanged, RecipientsChanged, and AppointmentTimeChanged events.
```

Comment:
I think we should either keep the code ticks or, better yet, add links to the new reference content for those events.

Reply:
@AlexJerabek We don't have any docs for those events yet so there's nothing to link to beside the eventtype enum. I'll add the ``.